projects/packages/post-list/src/class-post-list.php

```diff
@@ -80,43 +67,43 @@ public function enqueue_scripts( $hook ) {
 				plugin_dir_url( __DIR__ ) . 'build/style.rtl.css'
```

Comment:
I'm not sure this RTL stylesheet exists anymore. Are the columns reversed when we use an RTL language? If so, we might need to adjust the styles a bit to accommodate that.

Reply:
I'm honestly not sure what rtl.css is so I left it. I assumed maybe it was a compiled css file, something that was referenced post build. Sounds like we can remove it?

Reply:
Yeah I think you're right, but I'm not sure what happens to the post list with RTL languages. It might be worth activating one to see. If the order of the columns changes, then that could break this change.

Reply:
Oh! RTL is right-to-left written languages! So, this is a CSS file to make style changes based on if the language is RTL. I'm with you now. 😂 How do we test that? Change the browser language? I'll try it.

Reply:
Yeah, so we don't even that file in our project, so this probably does nothing at this point, right?

Reply:
That's right, sorry another acronym! You'd have to change the site/user language to an RTL one.

I think we just need to check it, and add a new CSS file if it's needed